robottelo/virtwho_utils.py

```diff
@@ -407,22 +410,23 @@ def virtwho_package_locked():
     assert "Packages are locked" in result[1]
 
 
-def create_http_proxy(name=None, url=None, type='https'):
+def create_http_proxy(name=None, url=None, http_type='https', org_name=DEFAULT_ORG):
```

Comment:
Only thing I would recommend here is to pass the `org` instance, and reference the field you need within this function (name, label, etc)

Reply:
Updated, thanks